core/tests/protractor_desktop/navigation.js

```diff
@@ -28,6 +30,10 @@ describe('Oppia static pages tour', function() {
     waitFor.pageToFullyLoad();
   });
 
+  it('Loads the page', async function() {
```

Comment:
Instead of making a new "it" for this, could you please integrate it instead in one of the existing tests? That would be a better way to illustrate, for new contributors, how Percy should be integrated.